src/app/cube/about-us/components/timeline/timeline.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+import {default as events} from './events.json';
+
+@Component({
+  selector: 'clark-timeline',
+  templateUrl: './timeline.component.html',
+  styleUrls: ['./timeline.component.scss']
+})
+export class TimelineComponent implements OnInit {
+
+  events = events;
```

Comment:
This looks like it should be deleted unless it is used in the timeline HTML file.